app/admin/admin_user.rb

```diff
@@ -28,17 +47,8 @@
   
   controller do
     def destroy
-      if AdminUser.true_admin? current_admin_user.id 
-        if current_admin_user.id.to_s == params[:id].to_s
-          flash[:alert] = "Warning: Site owner can not be deleted"
-          redirect_to "/admin/admin_users/"
-        else
-          super
-        end
-      else
-        flash[:alert] = selget_warning + " delete"
-        redirect_to "/admin/admin_users/"
-      end
+      destroy_user(params[:id])
+      redirect_to "/admin/admin_users/"
```

Comment:
Is it possible to change this to the helper routes function (i.e admin_user_path)?